### PR TITLE
Add myself (Youssef Soliman) to contributors.json

### PR DESCRIPTION
**Pull request by youssefsoli**
#### ba875c070aa875ed14de2baccdd64a35339dfcb5
<pre>
Add myself (Youssef Soliman) to contributors.json
<a href="https://bugs.webkit.org/show_bug.cgi?id=240084">https://bugs.webkit.org/show_bug.cgi?id=240084</a>

Reviewed by Jer Noble.

* metadata/contributors.json:

Canonical link: <a href="https://commits.webkit.org/250279@main">https://commits.webkit.org/250279@main</a>
git-svn-id: <a href="https://svn.webkit.org/repository/webkit/trunk@293806">https://svn.webkit.org/repository/webkit/trunk@293806</a> 268f45cc-cd09-0410-ab3c-d52691b4dbfc
</pre>

### CODE REVIEW

File: metadata/contributors.json
Comment (jernoble) [inline metadata/contributors.json]: You should omit this line. You're not quite a committer yet.

**Comment by jernoble** [comment]
r=me

**Review by jernoble — APPROVED** [review]

**Comment by JonWBedard** [comment]
> 

As Merge-Queue warned, commits need to be squashed before landing.

**Comment by JonWBedard** [comment]
The recently added commit looks like a merge-commit. What you want to do is reset your head with `git reset HEAD~3` and then re-commit this change.

**Comment by youssefsoli** [comment]
Figured it out finally (I hope)

**Comment by JonWBedard** [comment]
> Figured it out finally (I hope)

Looks good, sending through `merge-queue` now

**Pull request merged**
